client/components/navbar.js

```diff
@@ -164,9 +164,10 @@ class Navbar extends React.Component {
 /**
  * CONTAINER
  */
-const mapState = state => {
+const mapState = (state, ownProps) => {
   return {
-    isLoggedIn: !!state.user.id
```

Comment:
If you want to remove this, you might want to pass isLoggedIn from the route.js file because the render method in this component checks for that prop